dwave_networkx/drawing/qubit_layout.py

```diff
@@ -23,13 +23,16 @@
 from networkx import draw
 
```

Comment:
[line 16] "Tools to visualize Chimera lattices ..." --> change to "Tools to visualize QPU lattices ..."

Reply:
Should we not add a ``from dwave_networkx.drawing.qubit_layout import *`` line to the [init file](https://github.com/dwavesystems/dwave-networkx/blob/main/dwave_networkx/drawing/__init__.py)?

Reply:
No, I don't think we should.  These functions are for internal use.